app/src/main/java/swen_anigans/mathematicfanatic/Question.java

```diff
@@ -0,0 +1,71 @@
+package swen_anigans.mathematicfanatic;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * Created by Matthew Gallagher on 11/14/2016.
+ */
+
+public class Question implements Serializable {
+
+    private int firstNumber;
+    private int secondNumber;
+    private int submittedAnswer;
+    private int answer;
```

Comment:
If we have public getters and setters we should just make these variables public.